v3/plugins/redhat/vscode-tekton/0.1.0/meta.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+publisher: redhat-developer
+name: vscode-tekton
+version: 0.1.0
+type: VS Code extension
+displayName: Tekton
+title: Tekton pipelines support
+description: This extension adds rich pipeline support for Tekton
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-tekton
+category: Language
+firstPublicationDate: '2020-05-22'
+spec:
+  containers:
+    - image: "quay.io/ericwill/che-testing:latest"
+      name: vscode-tekton
+      memoryLimit: '512Mi'
+  extensions:
+    - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-yaml/vscode-yaml-0.7.2.vsix
+    - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-kubernetes-tools/vscode-kubernetes-tools-1.1.0.vsix
+    - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-tekton/vscode-tekton-pipelines-0.1.0.vsix
```

Comment:
0.2.0 is available https://github.com/redhat-developer/vscode-tekton/releases (btw there is a typo)
Could we use it?

Reply:
oh well 0.0.2 0.2.0 
![image](https://user-images.githubusercontent.com/407397/85577433-9a6fbc00-b60f-11ea-93a8-6d47a72e2d4f.png)

Reply:
0.0.2 seems to be a error but it is 0.2.0 (see tags and download links)